met-web/src/apiManager/endpoints/index.ts

```diff
@@ -0,0 +1,9 @@
+import { AppConfig } from '../../config';
+
+const API = {
+    MET_GET_ENGAGEMENTS: `${AppConfig.apiUrl}/engagement/`,
+    MET_GET_ENGAGEMENT: `${AppConfig.apiUrl}/engagement/<engagement_id>`,
+    MET_CREATE_ENGAGEMENT: `${AppConfig.apiUrl}/engagement/`,
+    MET_UPDATE_ENGAGEMENT: `${AppConfig.apiUrl}/engagement/`,
+};
```

Comment:
Should the above GET, CREATE and Update engagements be made into one URL. I feared confusion and it maybe being harder to change in the future if we wanted a different URL (for some reason) for them